skimage/data/__init__.py

```diff
@@ -327,6 +328,23 @@ def rocket():
     """
     return load("rocket.jpg")
 
+def satellite():
+    """Launch photo taken by satellite.
+
+    This is a satellite imagery showing an urban area
+
+    Notes
+    -----
+    This image was downloaded from USGS public data portal https://earthexplorer.usgs.gov/
```

Comment:
It is fine to include the copyright info link here, but it should explicitly be stated that these images are in the public domain.